README.md

```diff
@@ -67,10 +68,10 @@ This message is sent by a car to indicate that following must come to an end. Bo
 This message includes information about a leading vehicle and contains information relevant for a following car to be able to follow it. The LeaderStatus is sent in regular intervals of 125ms and does not expect a response.
 
 ***Fields***
-* uint32_t timestamp       - The time stamp (the time that the message has been sent) of the leading vehicle.
-* float  speed           - Current speed of the leading vehicle.
-* float steeringAngle    - Current steering angle of the leading vehicle.
-* uint8_t distanceTraveled - The distance travelled since the last status update.
+* uint64_t timestamp - The time stamp (the time that the message has been sent) of the leading vehicle, represented as UNIX Epoch time in milliseconds.
+* float speed - Current speed of the leading vehicle, the leading car should put its current Pedal Position Reading into this message field.
+* float steeringAngle - Current steering angle of the leading vehicle, the leading car should put its current Ground Steering Reading into this message field.
+* uint8_t distanceTraveled - The distance travelled since the last status update (according the odometer), represented in centimeters.
```

Comment:
wait, didn't we say that there was no odometer?

Reply:
There is no odometer, distanceTraveled will be whatever the sending car deems fit to put there, be it readings from the IMU or something else